### PR TITLE
Add myself (@BetterBlood) to the list of students

### LINKED ISSUE #24: Add myself (@BetterBlood) to the list of students
I would like to add myself to the list of students in the GitHub organization README.

### PR DESCRIPTION
**Pull request by BetterBlood**
Added myself to the list of students

### CODE REVIEW

**Comment by hadrylouis** [comment]
Could you please add a comment with the link to your issue with #xxx
You can follow the [create a pull request](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#create-a-pull-request) section of the course material

**Comment by BetterBlood** [comment]
> Could you please add a comment with the link to your issue with #xxx You can follow the [create a pull request](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#create-a-pull-request) section of the course material

Fixes #24

File: profile/README.md [hunk 1]
```diff
@@ -28,3 +28,4 @@ First name Last name [@GitHub username](https://github.com/USERNAME)
 - Alexandre Philibert [@AlexandrePhilibert](https://github.com/AlexandrePhilibert)
 - Valentin Ricard [@valentin-ricard](https://github-com/valentin-ricard) 
 - Jonas Troeltsch [@JonasTroeltsch](https://github.com/JonasTroeltsch)
+- Jeremiah Steiner [@BetterBlood](https://github.com/BetterBlood)
```
Comment (hadrylouis) [inline profile/README.md:37]: Could you please add yourself in alphabetical order (by last name)

**Review by hadrylouis — DISMISSED** [review]

File: profile/README.md (hunk 1 above)
Comment (BetterBlood) [reply profile/README.md:37]: Done

**Review by ludelafo — APPROVED** [review]

**Pull request merged**